Domybest (#1)
Kosuke-iga/SSS-master-thesis: doc/4_proposed.tex

\chapter{Proposed Scheme}\markright{Chapter 4: Proposed Scheme}\label{sec:proposed_scheme}

\section{Idea}

> Our goal is to design the defense scheme against each evasive technique on different targets, GSI and BD, to achive the practical CNN based IoT malware detection on IoT devices.

Reviewer: ```suggestion
Our goal is to design the defense scheme against each evasive technique on different targets, namely GSI and BD, to achieve practical CNN based IoT malware detection on IoT devices.

```

> In order to accomplish our goal, we propose an evasion-resilient IoT malware detection scheme with invalidating adversarial byte sequences and 1D convolutional filters.  

Reviewer: ```suggestion
In order to accomplish our goal, we propose an evasion-resilient IoT malware detection scheme with invalidating adversarial byte sequences and 1D convolutional filters.  

```
今更だけど、修論は著者一人だから、I にすべきとか言われるかも？？

> The main idea of our scheme is that there still remain regions which contribute to decision making in both manipulated targets after the attacks.

Reviewer: ```suggestion
The main idea of our scheme is that there still remain regions which contribute to decision making in both manipulated targets after the attacks.

```
この後にAMとOM内に、有用な場所が残存する理由を簡単に書くべき。
次のセクションで言ってるっぽいから、まとめる形で言えればいいかな。
例えば、AMやOMを作るのに、どちらにもある制約があるとか？

We aim to improve the detection accuracy by statically extracting/enhancing each valuable region of the target to the CNN model.

> 図 shows the manipulated area by adversaries and the area which have high degree of contribution for CNN to make decision on representative manipulated target.

Reviewer: ```suggestion
図 shows the manipulated area by adversaries and the area which have high degree of contribution for CNN to make decision on a representative manipulated target.

```
